crates/rolldown_debug/src/debug_data_propagate_layer.rs

```diff
@@ -76,3 +85,34 @@ where
     }
   }
 }
+
+#[derive(Debug)]
+pub struct ContextData(FxHashMap<String, String>);
+
+impl Deref for ContextData {
+  type Target = FxHashMap<String, String>;
+
+  fn deref(&self) -> &Self::Target {
+    &self.0
+  }
+}
+
+#[derive(Default)]
+pub struct ContextDataFinder {
+  context_data: FxHashMap<String, String>,
+}
+
+const CONTEXT_PREFIX: &str = "CONTEXT_";
+const CONTEXT_PREFIX_LEN: usize = CONTEXT_PREFIX.len();
+impl tracing::field::Visit for ContextDataFinder {
+  fn record_str(&mut self, field: &tracing::field::Field, value: &str) {
+    if field.name().starts_with(CONTEXT_PREFIX) {
+      let key = field.name()[CONTEXT_PREFIX_LEN..].to_string();
```

Comment:
/// Returns a string representing the name of the field.
    pub fn name(&self) -> &'static str {
        self.fields.names[self.i]
    }


`filed.name()` return `&'static str` I think you could use 'static str as your key, this could avoid unnecessary allocation.

Reply:
👌